src/systems.jl

```diff
@@ -183,6 +188,12 @@ for (Z, AZ) in ((:AffineContinuousSystem, :AbstractContinuousSystem),
                 return new{T, MT, VT}(A, c)
             end
         end
+        function $(Z)(A::Real, c::Real)
+             MA = Matrix{typeof(A)}(undef, 1, 1); MA[1, 1] = A
+             Vc = Vector{typeof(c)}(undef, 1); Vc[1] = c
```

Comment:
```suggestion
             Vc = [c]
```
I guess we could use Julia's convenience constructor for `Vector`s here as well, but we can also keep it as is. But then I suggest to write a helper function again.

```julia
julia> c = 1.0f32
1.0f32

julia> [c]
1-element Array{Float32,1}:
 1.0e32

julia> Vc = Vector{typeof(c)}(undef, 1); Vc[1] = c; Vc
1-element Array{Float32,1}:
 1.0e32

julia> @btime [$c]
  28.561 ns (1 allocation: 96 bytes)
1-element Array{Float32,1}:
 1.0e32

julia> @btime (Vc = Vector{typeof($c)}(undef, 1); Vc[1] = $c; Vc)
  30.418 ns (1 allocation: 96 bytes)
1-element Array{Float32,1}:
 1.0e32
```

Reply:
Here we can use `vcat(c)` :)

Reply:
> Here we can use `vcat(c)` :)

Is that better than `[c]`?

Reply:
It is identical.

Reply:
Here i would use either `[c]` or `vcat(c)`, i don't care which. The performance of the 3 methods considered is the same (try wrapping each into a function for `@btime`).

Reply:
If you do `@code_native` it give you exactly the same assembly instruction